### PR TITLE
add wk 15 s-g-c

### PR DESCRIPTION
**Pull request by lucyrose93**
@jsms90 
@bradreeder

### CODE REVIEW

File: stop-go-continue/fac-10/cohort-week-15 [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# STOP
+
+- #### Product owners who are not committed to participating (Edgar).
```
Comment (jsms90) [inline stop-go-continue/fac-10/cohort-week-15]: I don't think that bullet points and headings play well together (not sure)

Maybe just lose the bullet point?

**Comment by jsms90** [comment]
Thanks @bradreeder!

This formatting is quite hard to read though. Could we follow something like [this](https://github.com/foundersandcoders/london-curriculum/blob/master/stop-go-continue/fac-10/cohort-week-8.md) instead?

**Comment by jsms90** [comment]
@bradreeder bullets? :yum:

**Comment by bradreeder** [comment]
@jsms90 give me your best shot. ;)

**Comment by jsms90** [comment]
@bradreeder Already used that zing earlier today :exclamation: 

:stuck_out_tongue_closed_eyes:

**Pull request merged**